### DISCUSSION TITLE
Feature Request: Generate new palettes from a base color

### DISCUSSION THREAD

**Discussion by saaryab** [radix-ui/colors#54] [Ideas]
Hi, we love your color system but find ourselves wanting for a slightly different green option. Do you hand pick all the colors in each scale or do you have a standard set of levels so that given a single color (say step 9) you could generate the rest of the steps?

Lyft have a cool tool that does something like that but we are missing the parameters to recreate Radix compatible color scales: https://lyft-colorbox.herokuapp.com/

I'd be happy to help build a tool like that if you guys can help with the math :-)

**Comment by mstade** [comment]
I found this tool which is pretty cool, and has a Radix preset: https://grayscale.design/app

Here's an example with adding #ff00ff as a color: <https://grayscale.design/app?lums=98.10,96.40,90.10,83.30,75.60,65.80,52.60,37.40,15.20,12.80,10.20,1.20&palettes=%23ff00ff&filters=0%7C0&names=&labels=>

I've played around with it for a while, results seem pretty good. One fun thing is to use something like https://huemint.com/ to find some nice colors, then plug them into the aforementioned app and hey presto: new scales!

**Comment by saaryab** [comment]
Thanks @mstade ! That's a great find. I'll be sure to try it out.

**Comment by mrmartineau** [comment]
Pretty sure this can be closed now since https://www.radix-ui.com/colors/custom exists..
